6.0.0 (#90)
MetaMask/eth-json-rpc-filters: CHANGELOG.md

## [Unreleased]

## [6.0.0]
### Changed

> - Fix: changes in subscription middleware to not throw error if block for a block number is not found ([#89](https://github.com/MetaMask/eth-json-rpc-filters/pull/89))

Reviewer: Our breaking change entries conventionally start with `**BREAKING:** `

Also maybe better to describe this as "filter middleware", as they can be used independently of the subscriptions as well. Here is a suggested rewording:
```suggestion
- **BREAKING:** Block filter middleware will not throw an error if a block is not found ([#89](https://github.com/MetaMask/eth-json-rpc-filters/pull/89))
```
